### PR TITLE
[vividus] Fix handling of an attempt to run empty suite

### PR DESCRIPTION
**Pull request by valfirst**

### CODE REVIEW

File: vividus/src/main/java/org/vividus/runner/StoriesRunner.java [hunk 1]
```diff
@@ -51,6 +51,10 @@ public StoriesRunner()
     public void run()
     {
         Map<String, List<String>> paths = getPaths();
+        if (paths.isEmpty())
+        {
+            throw new InvalidConfigurationException("No batches with tests to execute are configured");
```
Comment (ikalinin1) [inline vividus/src/main/java/org/vividus/runner/StoriesRunner.java:56]: EmptySuites(Batches)Exception?

Can we consider the selenium way  https://www.selenium.dev/exceptions/

**Review by uarlouski — APPROVED** [review]

File: vividus/src/main/java/org/vividus/runner/StoriesRunner.java (hunk 1 above)
Comment (uarlouski) [reply vividus/src/main/java/org/vividus/runner/StoriesRunner.java:56]: just validate.isTrue?)

Comment (valfirst) [reply vividus/src/main/java/org/vividus/runner/StoriesRunner.java:56]: > EmptySuites(Batches)Exception

I plan the generic exception will be reused.

> Can we consider the selenium way https://www.selenium.dev/exceptions/

No references to docs (yet) - too many potential issues, so again no reason to make this exception very specific.

> just validate.isTrue?)

`IllegalArgumentException` is too generic and used too often

Comment (uarlouski) [reply vividus/src/main/java/org/vividus/runner/StoriesRunner.java:56]: I bet end users won't check the exception name

Comment (valfirst) [reply vividus/src/main/java/org/vividus/runner/StoriesRunner.java:56]: Some specific exception is easier to find it in the docs

**Review by ikalinin1 — APPROVED** [review]

**Pull request merged**